src/tools.js

```diff
@@ -6,20 +6,23 @@ const processingFunctions = {};
 export function processSingleResultResponse(objectType, result, url) {
   const ValueObject = valueObjects[objectType];
   const definition = valueObjectDefinitions[objectType];
-
   const processedResult = {};
 
   for (const item in definition) {
-    if (!definition.hasOwnProperty(item)) {
-      continue;
-    }
 
     if (definition[item] === null) {
-      // Just copy.
-      processedResult[item] = result[item];
+      if (result.hasOwnProperty(item)) {
+        // Copy value from 'result':
+        processedResult[item] = result[item];
+      } else if(result.properties && result.properties.hasOwnProperty(item)) {
```

Comment:
Er is in tools.js ook al processFeatureCollection, voor een lijst van dit soort geo-results. Deze parcels hebben type: 'Feature'. Misschien ipv in deze functie alle mogelijkheden afvangen een processFeature voor de single results met properties?